components/IndoorDirections/IndoorScenario.js

```diff
@@ -0,0 +1,67 @@
+import React, { useEffect } from 'react';
+import { G } from 'react-native-svg';
+import { HallXCoordinates } from '../../constants/HallXCoordinates';
+import { ClassGraph } from '../../constants/ClassGraph';
+import { DifferentFloorDirections } from './TypesOfDirections/DifferentFloorDirections';
+import { SameFloorDirections } from './TypesOfDirections/SameFloorDirections';
+import { DifferentBuildingDirections } from './TypesOfDirections/DifferentBuildingDirections';
+
+export function IndoorScenario({floor}) {
+    const rooms = HallXCoordinates();
+    const graph = ClassGraph();
+    const routeToTake = whichPathToTake("H820", "MB3.125");
+
+    switch (routeToTake) {
+        case "SAME_FLOOR":
+            return <SameFloorDirections rooms={rooms} floor={floor} from={"H855"} to={"H829"}/>
+            break;
+        case "DIFFERENT_FLOOR":
+            return <DifferentFloorDirections rooms={rooms} floor={floor} from={"H855"} to={"H521"}/>;
+            break;
+        case "DIFFERENT_BUILDING":
+            return <DifferentBuildingDirections rooms={rooms} floor={floor} from={"H855"} to={"MB3.125"}/>;
+            break;
+    }
+
```

Comment:
Is it normal to have the room number hard coded in the *from* and *to* variables ?

Reply:
Oh man, this is the wrong commit! Good spot. I will fix the commit now.